doc/help/kdb-namespace.md

```diff
@@ -0,0 +1,44 @@
+# kdb-namespace(1) -- Get the namespace of a key name
+
+## SYNOPSIS
+
+`kdb namespace <key name>`
+
+Where `key name` is the name of the key.
+
+## DESCRIPTION
+
+This command is used to retrieve the namespace of a key, including the eventual trailing ':'.
+For the cascading namespace, the empty string is returned.
+
+## RETURN VALUES
+
+This command will return the following values as an exit status:
+
+- 0:
+  No errors.
+- 7:
+  invalid key name passed, see [kdb(1)](kdb.md) for a list of return codes used by kdb.
+
+## OPTIONS
+
+- `-n`, `--no-newline`:
+  Suppress the newline at the end of the output.
+- `-S`, `--no-separators`:
```

Comment:
Is not needed, see https://github.com/ElektraInitiative/libelektra/issues/4012#issuecomment-909515824

Reply:
I have removed it.